apps/builder/app/builder/features/sidebar-left/panels/pages/settings.tsx

```diff
@@ -1,1076 +0,0 @@
-import { z } from "zod";
```

Comment:
was there so much changes so git cannot determine rename?

Reply:
yeah, not sure, seems like it should have renamed using git mv first

Reply:
very weird

Reply:
I can try merge a separate pr that will rename that file first so we can see here a proper diff